### PR TITLE
Delete unused code

### PR DESCRIPTION
**Pull request by omarelmalki**
Delete unused classes, xml files and UI elements.

### CODE REVIEW

File: app/src/androidTest/java/com/github/brugapp/brug/QrCodeShowActivityTest.kt [hunk 1]
```diff
@@ -8,23 +8,25 @@ import androidx.test.espresso.assertion.ViewAssertions.matches
...
+import com.github.brugapp.brug.model.MyItem
+import com.github.brugapp.brug.model.MyUser
 import com.github.brugapp.brug.ui.QrCodeShowActivity
 import org.junit.Test
 import org.junit.runner.RunWith
 
 @RunWith(AndroidJUnit4::class)
 class QrCodeShowActivityTest{
-    private val item = Item("Phone","Samsung Galaxy S22","23")
+    private val item = MyItem("Phone", ItemType.Phone.ordinal,"my phone", false)
```
Comment (pmcintyr) [inline app/src/androidTest/java/com/github/brugapp/brug/QrCodeShowActivityTest.kt:20]: Good job here on replacing the old local types Item & User with MyItem and MyUser in these tests

File: app/src/main/java/com/github/brugapp/brug/ui/MapBoxActivity.kt [hunk 2]
```diff
@@ -124,15 +124,15 @@ class MapBoxActivity : AppCompatActivity() {
 
                         val viewAnnotationManager = binding.mapView.viewAnnotationManager
                         val viewAnnotation = viewAnnotationManager.addViewAnnotation(
-                            resId = R.layout.sample_hello_world_view,
+                            resId = R.layout.on_item_map_click_view,
```
Comment (pmcintyr) [inline app/src/main/java/com/github/brugapp/brug/ui/MapBoxActivity.kt:127]: If I understand correctly this is just a rename, am I correct?

File: app/src/main/res/layout/activity_sign_in.xml [hunk 3]
```diff
@@ -57,7 +57,7 @@
         android:text="@string/demo_text"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.25"
+        app:layout_constraintHorizontal_bias="0.498"
```
Comment (pmcintyr) [inline app/src/main/res/layout/activity_sign_in.xml:60]: Did you just recenter this demo text?

File: app/src/main/java/com/github/brugapp/brug/ui/MapBoxActivity.kt (hunk 2 above)
Comment (omarelmalki) [reply app/src/main/java/com/github/brugapp/brug/ui/MapBoxActivity.kt:127]: Yes!

File: app/src/main/res/layout/activity_sign_in.xml (hunk 3 above)
Comment (omarelmalki) [reply app/src/main/res/layout/activity_sign_in.xml:60]: Yes

**Review by ha2san — APPROVED** [review]
Great job, the code is clearer now

**Review by hamzahass — APPROVED** [review]
Thanks for cleaning the project, not much to say. However, I just wanted to know why we kept MyUser and MyItem, and deleted User and Item ?

**Comment by omarelmalki** [comment]
> Thanks for cleaning the project, not much to say. However, I just wanted to know why we kept MyUser and MyItem, and deleted User and Item ?

Because MyItem and MyUser were used in the code and most of the tests.

**Pull request merged**